### ISSUE TITLE
Option to also retire mappings when retiring a concept

### ISSUE THREAD

**Issue by paynejd** [OpenConceptLab/ocl_issues#1828]
In a source, when I retire a concept that also has mappings in the same source, I want the TermBrowser to prompt me to optionally also retire the associated mappings.

**Comment by snyaggarwal** [comment]
Will this also utilise $cascade level=0?

**Comment by jamlung-ri** [comment]
> Will this also utilise $cascade level=0?

@snyaggarwal Could you clarify your question for me? I was thinking that cascade level 0 only returns the requested concept. Does it actually return both the concept and its mappings?

**Comment by snyaggarwal** [comment]
@jamlung-ri You are right, I meant level 1

**Comment by jamlung-ri** [comment]
Yeah I do think it'll leverage Cascade level 1, but only filter down to the mappings to perform retirement on. The concepts returned by cascade level 1 should keep their existing retirement status.
